tests/test_contrib/test_classifier/test_boundaries.py

```diff
@@ -320,9 +326,7 @@ def test_fit_draw_poof(self):
         viz.draw.assert_called_once_with(X_two_cols, y)
         viz.poof.assert_called_once_with()
 
-    @pytest.mark.xfail(
-        sys.platform == 'win32', reason="images not close on windows"
-    )
+    @pytest.mark.xfail(reason="numpy structured arrays have changed since v1.14")
```

Comment:
Couldn't find a solution for this test, so just marked as xfail since this is a contrib package. If we move this back into our classifier module, we'll either have to find a solution for structured arrays or simply not support them.